nothing sets this to 0, but for some reason it gets undef'd
diff --git a/build/cmake/setup.h.in b/build/cmake/setup.h.in
index d4ef1fc..69ecd6c 100644
--- a/build/cmake/setup.h.in
+++ b/build/cmake/setup.h.in
@@ -869,7 +869,7 @@
 /*
  * Define if large (64 bit file offsets) files are supported.
  */
-#cmakedefine HAVE_LARGEFILE_SUPPORT 1
+#define HAVE_LARGEFILE_SUPPORT 1
 
 /*
  * Use OpenGL
